crates/solvers-dto/src/auction.rs

```diff
@@ -165,6 +329,94 @@ pub enum Liquidity {
     LimitOrder(ForeignLimitOrder),
 }
 
+// todo: Currently, it strictly follows the manual api schema. This has to be
+// automated and deleted.
```

Comment:
Is this something blocking this PR or just future work to make working with this code more ergonomic?

Reply:
This PR aims to automate the OpenAPI schema generation without altering its structure. In separate PRs, it would be good to try to remove manual `ToSchema` implementations, but this will change the schema structures.